Steps/PrGate.yml

```diff
@@ -87,12 +87,17 @@ steps:
 
 # Trim the package list if this is a PR
 - ${{ if eq(parameters.do_pr_eval, true) }}:
+  - powershell:

```

Comment:
nit: :)

Reply:
Sorry, not following what the nit is about here